src/ucp/core/ucp_request.inl

```diff
@@ -166,6 +166,7 @@ ucp_request_try_send(ucp_request_t *req, ucs_status_t *req_status,
 {
     ucs_status_t status;
 
+    /* coverity[address_free] */
```

Comment:
can you pls comment what was the issue here?

Reply:
*(*req).send.uct.func" frees address of "(*req).send.uct when func is resolved to test_uct_pending::pending_send_op_ok.
but then, if the req is released, then UCS_OK should return and then req isn't accessed anymore.

Reply:
let's add comment here that coverity wrongly resolves (*req).send.uct.func to test_uct_pending::pending_send_op_ok